mrtt-ui/src/components/ProjectDetailsForm.js

```diff
@@ -183,13 +186,9 @@ const ProjectDetailsForm = () => {
         <FormQuestionDiv>
           <FormLabel>1.3 What is the overall site area?</FormLabel>
         </FormQuestionDiv>
-        {isError && (
-          <Typography variant='subtitle' sx={{ color: 'red' }}>
-            Submit failed, please try again
-          </Typography>
-        )}
+        {isError && <ErrorText>{language.error.submit}</ErrorText>}
```

Comment:
Al is gunna decide he doesnt like the wording of stuff later, and this makes it easier for him to change it up.